resolve module installation error in 17.0

manishkumarbohra opened the pull request.

manishkumarbohra committed:
resolve module installation error in 17.0

pedrobaeza commented:
Thanks for the contribution.

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

manishkumarbohra commented:
ok, thanks @pedrobaeza

manishkumarbohra committed:
module migration from 16.0 to 17.0
1. hr_employee_birth_name
2. hr_employee_birthday_mail
3. hr_employee_digitized_signature
4. hr_employee_document
5. hr_employee_medical_examination
6. hr_employee_partner_external
7. hr_employee_phone_extension
8. hr_employee_relative
9. hr_employee_ssn

#1291

manishkumarbohra commented:
@pedrobaeza  i will be close this PR request and submit the new one based on community guideline. 

thanks for sharing all the details.
